core/imodel-bridge/src/BADGERSIssueReporter.ts

```diff
@@ -0,0 +1,288 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+import { DbResult, Guid, GuidString } from "@bentley/bentleyjs-core";
+import { ECDb } from "@bentley/imodeljs-backend";
+import { BridgeIssueReporter } from "./BridgeIssueReporter"
+import * as fs from "fs";
+import * as path from "path";
+import * as os from "os";
+import { fips } from "crypto";
+
+interface ContextInfo {
+    activityId: string;
+    contextId: string;
+    iModelId: string;
+    briefcaseId: number;
+    operationType?: string;
+    jobId?: string;
+}
+
+interface FileInfo {
+    fileId?: string;
+    filePath?: string;
+}
+
+/** BADGERS issue reporter. Can be provided to BridgeRunner in setIssueReporter
+ * @beta
+ */
+export class BADGERSIssueReporter implements BridgeIssueReporter {
```

Comment:
Unbelievably nitpicky but can we ditch the hideous ALL-CAPS?

Reply:
Also, @pmconne, is our indentation standard still 2 spaces?

Reply:
yes.

Reply:
The name and the indentation has been fixed